model/godleyTable.cc

```diff
@@ -169,6 +169,11 @@ GodleyTable::AssetClass GodleyTable::_assetClass
   return _assetClass(col);
 }
 
+bool GodleyTable::singleEquity() {
+	GodleyTable::AssetClass cls=GodleyAssetClass::equity;
+	if (m_assetClass[cols()-1]==cls && m_assetClass[cols()-2]==cls) return false;
```

Comment:
Is OK, but I would write this as:
return m_assetClass[cols()-2]!=GodleyAssetClass::equity

If the second to right most column is equity, then so will the right most column.
The only other thing to consider is if the table could ever have fewer than 3 columns. I'd probably add that as an assertion.

Reply:
Done, thanks :)